iBridges.py

```diff
@@ -227,11 +224,10 @@ def main():
     utils.utils.init_logger(THIS_APPLICATION)
```

Comment:
The idea here is to initialize the logging, instantiate the Context singleton, and immediately set the log level in case it is not the default.  That way all the logging it done as expected.